src/components/Textarea/Textarea.test.tsx

```diff
@@ -1,37 +1,33 @@
 import { render, screen } from "@testing-library/react";
-import { mount, shallow } from "enzyme";
 import React from "react";
 
 import Textarea from "./Textarea";
 
 describe("Textarea ", () => {
   it("renders", () => {
-    const wrapper = shallow(<Textarea id="test-id" />);
-    expect(wrapper).toMatchSnapshot();
+    render(<Textarea id="test-id" wrapperClassName="textarea" />);
+    // Get the wrapping element.
+    // eslint-disable-next-line testing-library/no-node-access
+    expect(document.querySelector("textarea")).toMatchSnapshot();
```

Comment:
Use a test id, or container.

Reply:
Done. Though, just to be clear, using the container requires disabling an extra rule `// eslint-disable-next-line testing-library/no-node-access, testing-library/no-container`.